dev-scripts/local_docker_create_images_and_tag.sh

```diff
@@ -2,7 +2,7 @@
 
 export DeveloperBuild="true"
 
-currentDir = `pwd`
+currentDir=`pwd`
```

Comment:
BTW you may use https://www.shellcheck.net/ to get other review on shell scripts.